app/partials/errorDetail.html

```diff
@@ -1,9 +1,12 @@
 <pagination>
 
 <div class="detail-header content-l">
-    <div class="content-l_col content-l_col-2-3">
+    <div class="content-l_col content-l_col-1-3">
         <pagination-size></pagination-size>
     </div>
+    <div class="content-l_col content-l_col-1-3">
+        <code-descriptions></code-descriptions>
+    </div>
```

Comment:
This is just going to split that row into 1/3s and place the help in the center column. Wouldn't it look better to have the code-descriptions button next to the Export button?